### ISSUE TITLE
New session token doesn't update the Firefox container session

### ISSUE THREAD

**Issue by michaelfecher** [Systanics/CloudGlance#22]
**Describe the bug**

**To Reproduce**
1. Session of a profile (profileName in the example `md-kdev`) is expired in the Firefox container. 
Don't close the container tab.
[![](https://i.imgur.com/h8GW1xKm.jpg)](https://i.imgur.com/h8GW1xK.png)
3. Switch to CloudGlance profiles
4. Spot the `md-kdev` profile and click on "Get new Token"
[![](https://i.imgur.com/uYzciB4m.jpg)](https://i.imgur.com/uYzciB4.png)
6. Switch back to the container tab of the profile `md-kdev` and hit F5 or click "Reload"
7. Redirect on AWS Login page
[![](https://i.imgur.com/Y3wXBJIm.jpg)](https://i.imgur.com/Y3wXBJI.png)

**Expected behavior**
After Step 6 I expected to continue where I have left off in my selected profile.
Instead, the new token for the session seems to get lost

**Desktop (please be as specific as possible):**
Windows 10

**App version (click on the "Boundaries" menu item or at the bottom of the settings page)**
Version 0.0.113

**Comment by rehanvdm** [comment]
It should not be doing that, I will see if I can recreate and fix.

**Comment by rehanvdm** [comment]
Actually reading this properly this time😅. There is nothing Cloud Glance can do about this, Firefox does not have supported methods to manipulate tabs from the CLI. I can't find anything to list open tabs + indicate their container and then either refresh or close and open them again. It would be really nice to have but the way FF and AWS work after a token expired does not allow this.
